app/scripts/ui.js

```diff
@@ -40,10 +41,29 @@ const PHISHING_WARNING_PAGE_TIMEOUT = ONE_SECOND_IN_MILLISECONDS;
 const PHISHING_WARNING_SW_STORAGE_KEY = 'phishing-warning-sw-registered';
 
 let extensionPort;
+let traceContext;
 
 start().catch(log.error);
 
 async function start() {
+  const startTime = performance.now();
+
+  traceContext = trace({
+    name: 'UI Startup',
```

Comment:
In retrospect, probably worth using an enum for these to avoid any risk of mistyping.

Reply:
I'm 👍  on adding in enum in this PR.

Reply:
Done.